theia-flutter-docker/latest.package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "private": true,
+    "theia": {
+        "frontend": {
+            "config": {
+                "applicationName": "Theia Flutter Example",
+                "preferences": {
+                    "files.enableTrash": false
+                }
+            }
+        }
+    },
+    "dependencies": {
+        "@theia/editor-preview": "latest",
+        "@theia/file-search": "latest",
+        "@theia/getting-started": "latest",
+        "@theia/git": "latest",
+        "@theia/json": "latest",
+        "@theia/markers": "latest",
+        "@theia/messages": "latest",
+        "@theia/navigator": "latest",
+        "@theia/outline-view": "latest",
+        "@theia/plugin-ext-vscode": "latest",
+        "@theia/preferences": "latest",
+        "@theia/preview": "latest",
+        "@theia/search-in-workspace": "latest",
+        "@theia/terminal": "latest",
+        "@theia/textmate-grammars": "latest"
+    },
+    "devDependencies": {
+        "@theia/cli": "latest",
+        "@theia/debug": "latest"
+    },
+    "resolutions": {
+        "vscode-languageserver-protocol": "3.15.0-next.9",
+        "vscode-languageserver-types": "3.15.0-next.5",
+        "**/vscode-json-languageserver/**/vscode-languageserver": "6.0.0-next.1"
+    },
+    "scripts": {
+        "postinstall": "download-debug-adapters"
+    },
+    "adapterDir": "plugins",
+    "adapters": {
+        "dart-code": "https://ms-vscode.gallery.vsassets.io/_apis/public/gallery/publisher/Dart-Code/extension/dart-code/3.2.1/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage",
```

Comment:
This extension is licensed under MIT, which is good. However it's our understanding that the Visual Studio Marketplace's terms of service do not allow non-VS Code products to use it. So we would need to find this `.vsix` elsewhere. 

On GH in their [repo's Releases](https://github.com/Dart-Code/Dart-Code/releases), they have the `.vsix` for some releases (looks to be mostly beta / test releases unfortunately) - maybe you can pick one there that seems to work ok and use the URL instead of the current one? 

As a last resort, we have forked some extensions repo under the `theia-ide` org, build from sources and made a specific `.vsix` available in our fork's releases tab.

Reply:
I was just mimicking what the dart gitpod container was doing, but sounds good

Reply:
> I was just mimicking what the dart gitpod container was doing, but sounds good

I expect they'll do the same soon - they have the same info regarding the marketplace terms-of-use.